integration_test/integration_test.go

```diff
@@ -0,0 +1,322 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"text/template"
+	"time"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	compute "google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+)
+
+var (
+	commit = flag.String("commit", "", "git commit to test")
+	runID  = time.Now().Unix()
+)
+
+const (
+	cloudScope        = "https://www.googleapis.com/auth/cloud-platform"
+	monitorWriteScope = "https://www.googleapis.com/auth/monitoring.write"
+	// benchFinishString should keep in sync with the finish string in busybench.
+	benchFinishString = "busybench finished profiling"
+)
+const startupTemplate = `
+#! /bin/bash
+
+#Fail on any error
+set -e pipefail
+
+# Display commands being run
+set -x
+# Install git
+sudo apt-get update
+sudo apt-get -y -q install git-all build-essential
+
+# Install desired version of Node.js
+curl -sL https://deb.nodesource.com/setup_{{.NodeVersion}}.x -o nodesource_setup.sh
+sudo bash nodesource_setup.sh
+sudo apt-get install nodejs
+npm -v
+node -v
+
+# Install agent
+git clone https://github.com/GoogleCloudPlatform/cloud-profiler-nodejs.git 
+cd cloud-profiler-nodejs
+git reset --hard {{.Commit}}
+npm install
+npm run compile
+npm pack
+VERSION=$(node -e "console.log(require('./package.json').version);")
+PROFILER="$HOME/cloud-profiler-nodejs/google-cloud-profiler-$VERSION.tgz"
+
+TESTDIR="$HOME/test"
+mkdir -p "$TESTDIR"
+cp "e2e/busybenchmark.js" "$TESTDIR"
+cd "$TESTDIR"
+
+npm install "$PROFILER"
+
+# Run benchmark with agent
+GCLOUD_PROFILER_LOGLEVEL=5 GAE_SERVICE={{.Service}} node --require @google-cloud/profiler busybenchmark.js 600
+
+# Indicate script finished
```

Comment:
Indicate to who? The comment is not useful as is. You should probably explain why it's here, not what it's doing (what it's doing is clear from the code).